eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -42,44 +39,41 @@ parameters:
   type: object
   default:
     Public:
-      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-test)
```

Comment:
@Luyunmt did this revert back to `-test` ?

Reply:
Yes

Reply:
It needs to be without that suffix. The `-test` keyvaults were for testing and are not maintained anymore (and will be deleted).